### ISSUE TITLE
Version mismatches between tags and npm packages for 1.11.x

### ISSUE THREAD

**Issue by matthewhanson** [nasa/cumulus#999]
There appears to be a mismatch between the tags for at least versions 1.11.2 and 1.11.3 and the same versions published to npm.

Cloned, checked out 1.11.2, 1.11.3, 1.12.0, and 1.12.1, then installed from npm and used weld to compare the directories.

The differences in the 1.12.x versions were as expected - no test directories in the npm package, and other minor differences likely due to packaging (I didn't look in depth across all).

The differences in the 1.11.x tag versions and their npm package counterparts however were substantial.

Just showing an example screenshot of the API for 1.11.2

![image](https://user-images.githubusercontent.com/1416900/57964289-c8f57080-7900-11e9-839a-935dc4bb966c.png)

Blue shows file differences, and when examined these are significant differences in the code. There's also new files that have been added.  All the sub-packages I looked at had similar differences.

This came up as I was using 1.11.3 and wanted to use a custom cloudformation.template.yml file and ran into problems until I realized there was a difference between the version I was using (the 1.11.3 tagged version in the repo) and the original version I was using in my deployment that was from npm.

**Comment by markdboyd** [comment]
Hey @matthewhanson - I think this issue is related to a recent discovery we made that our NPM packages were not being published to pin their dependencies to specific versions, which means that you would get installed package sub-dependencies that were incompatible. 

Can you take a look at #970 and see if it seems related or like it would fix your issue? If so, that fix is included in the next release that will go out today or tomorrow.

**Comment by matthewhanson** [comment]
It certainly seems like that issue could have caused this problem, so that's great it's getting fixed.  The 1.12.X versions didn't seem to be affected, but I didn't look super close so they might have.

But now that they are fixed the bigger problem is that the NPM published packages are not the same as what is tagged on GitHub for past versions, and there seems no easy way to fix this. You can't republish NPM packages with the same version ids, and recreating all the tags seems a mess - not even sure how you would do it.

The problem for users comes in if someone tries to do what I did: wanting to use a customized version of a config that is supplied with Cumulus.  So you go to the tagged version on GitHub and use that as a template, but there's differences and they might be small and could go unnoticed. 

Since this is only for previous versions, it doesn't really seem worth it to fix, but it may be worth a note in the documentation for affected versions that the GitHub tags do not match the NPM published versions.

**Comment by markdboyd** [comment]
@matthewhanson Ah, I think I understand better now. So your issue was basic code incompatibility between the published NPM version and the repo tag. I know that's exactly what you said, but for some reason I just got it. I thought the issue related to the Cumulus dependencies of published Cumulus NPM packages, hence the PR I referenced, but that does not seem to be the case. Sorry for the confusion. 

I don't understand how we were able to publish NPM versions with different code than the repo tag, but maybe we can just document this somewhere for future reference. Thanks for the heads up 👍
